ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -144,6 +153,8 @@ _init_modes (bounding_boxes * bdata)
     return TRUE;
   } else if (bdata->mode == TF_SSD_BOUNDING_BOX) {
     return TRUE;
+  } else if (bdata->mode == OV_PERSON_DETECTION_BOUNDING_BOX) {
+    return TRUE;
   }
   return TRUE;
```

Comment:
Always return TRUE?

Reply:
@gichan-jang As you pointed out, the lines at 156 ~ 157 and at 195 ~ 196 are not necessary. I will remove them.